docs/Project-Configuration/Best-Practices.md

```diff
@@ -48,4 +48,4 @@ echo $foo
 
 ## Distro Choice
 
-Tasks on more popular distros are often run quicker than tasks on less popular ones. Prefer more popular distros where possible.
+Tasks on more popular distros are often run quicker than tasks on less popular ones. Prefer more popular distros where possible. For more information about avaliable distros choices see [Guidelines around Evergreen distros](https://wiki.corp.mongodb.com/display/DBDEVPROD/Guidelines+around+Evergreen+distros)
```

Comment:
nit: i always recommend linking to the hashed version of the wiki page (in this case, it's: https://wiki.corp.mongodb.com/x/CZ7yBg)  so that if the title ever changes, you don't need to update this link here. (I got this from clicking the "Share" button on the top right of the Confluence Wiki page.)

Reply:
Updated in a new commit